### PR TITLE
Free trial explanation

### PR DESCRIPTION
**Pull request by danjohansenconsulting**
New Cloud docs page to describe the new Free Trial plan that was just launched.

This is a draft and now open for comment review, with expectation of deploying end-of-week.  I still need to convert all of the included links to on-links, but the rest of the page should be considered a release candidate.

### CODE REVIEW

File: docs/guides/cloud/free-trial.mdx [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+- [Enterprise Reporting](/guides/cloud/analytics/enterprise-reporting): Spot trends and anomalies,
+  track usage, and govern your overall test suites across all projects in your organization.
+- [Data Extract API](/guides/cloud/integrations/data-extract-api): Automatically download and extract
+  test data from Cypress Cloud, allowing endless possibilities for automating your SDLC workflows and
+  generating custom reporting.
+
+## Free trial onboarding process
+
+When you begin a free trial with Cypress cloud you will notice a progress indicator in the
+navigation sidebar that helps you gain understanding of the features and capbilities of Cloud.
```
Comment (MikeMcC399) [inline docs/guides/cloud/free-trial.mdx:118]:
```suggestion
navigation sidebar that helps you gain understanding of the features and capabilities of Cloud.
```

**Comment by MikeMcC399** [comment]
The screenshot `static/img/guides/cloud/free-trial/trial-progress-bar.jpg` contains the word "setup" used as a verb. It should be "set up" (see https://www.merriam-webster.com/dictionary/setup - "setup" is the noun, "set up" is the verb).

File: docs/guides/cloud/free-trial.mdx [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+- Integrations for On-Prem Instances: Seamlessly integrate with your existing infrastructure.
+- [Enterprise Reporting](/guides/cloud/analytics/enterprise-reporting): Spot trends and anomalies,
+  track usage, and govern your overall test suites across all projects in your organization.
+- [Data Extract API](/guides/cloud/integrations/data-extract-api): Automatically download and extract
+  test data from Cypress Cloud, allowing endless possibilities for automating your SDLC workflows and
+  generating custom reporting.
+
+## Free trial onboarding process
+
+When you begin a free trial with Cypress cloud you will notice a progress indicator in the
```
Comment (MikeMcC399) [inline docs/guides/cloud/free-trial.mdx:117]:
```suggestion
When you begin a free trial with Cypress Cloud you will notice a progress indicator in the
```

**Comment by danjohansenconsulting** [comment]
@MikeMcC399 agreed on "setup" vs "set up".  There are some copy and styling adjustments that will be made to that page in Cloud and once that is done I'll make sure we update the screen cap here.

File: docs/guides/cloud/free-trial.mdx [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+and simply revert to our _Starter_ plan. The _Starter_ plan is free, but does not have the same
+set of features. The _Starter_ plan will also limit data retention to 30 days and test results
+to 500 per monthly period. Details of the _Starter_ plan are available on the
+[pricing page](https://www.cypress.io/pricing).
+
+:::caution
+
+<Icon name="exclamation-triangle" /> The only restriction during a free trial is
+that you are not allowed to [transfer](/guides/cloud/account-management/projects#Transfer-ownership)
+existing Cypress Cloud projects into your free trial organization.{' '}
```
Comment (MikeMcC399) [inline docs/guides/cloud/free-trial.mdx:37]: Is this necessary at the end of the line?

`{' '}`

File: docs/guides/cloud/free-trial.mdx [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+Experience enhanced testing efficiency with Cypress Cloud’s
+[Smart Orchestration](/guides/cloud/smart-orchestration/overview) capabilities.
+Seamlessly [parallelize tests](/guides/cloud/smart-orchestration/parallelization) for faster
+execution, [prioritize failed specs](/guides/cloud/smart-orchestration/spec-prioritization) to
+address critical issues promptly, and automate
+[test run cancellation](/guides/cloud/smart-orchestration/run-cancellation) based on custom
+thresholds. Intelligent test coordination, powered by Cypress Cloud, ensures you get the most out
+of your testing efforts.
+
+Additionally, with Cypress 13, you can leverage [Test Replay](/guides/cloud/test-replay), our
```
Comment (MikeMcC399) [inline docs/guides/cloud/free-trial.mdx:87]: referred to as "v13 and up" on https://docs.cypress.io/guides/cloud/test-replay

File: docs/guides/cloud/free-trial.mdx [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+This is the same list of Cloud features available in our Enterprise plan as described on the
+[Cypress Cloud pricing page](https://www.cypress.io/pricing). 50,000 test results are included!
+
+Experiencing the capabilities of Cypress Cloud during a trial is the quickest way to realize
+that your testing life wouldn't be the same without it! After starting your trial you can select
+and subscribe to a Cypress Cloud plan at any time.
+
+However, if you determine that subscribing to a Cloud plan is
+not workable at the end of 14 days, rest assured that your account will remain available to you
+and simply revert to our _Starter_ plan. The _Starter_ plan is free, but does not have the same
```
Comment (jennifer-shehane) [inline docs/guides/cloud/free-trial.mdx:28]:
```suggestion
and revert to our _Starter_ plan. The _Starter_ plan is free, but does not have the same
```

File: docs/guides/cloud/free-trial.mdx (hunk 4 above)
Comment (jennifer-shehane) [inline docs/guides/cloud/free-trial.mdx:87]: I feel like this should be the first thing on the list and have its own section. This is the most valuable feature of the Cloud imo. It'd be great to have a gif/video here

Comment (danjohansenconsulting) [reply docs/guides/cloud/free-trial.mdx:87]: agreed.  Next commit will move Test Replay into an isolated section, and it is moved to the top of the list.  The YouTube demo embed will be included.

Comment (danjohansenconsulting) [reply docs/guides/cloud/free-trial.mdx:87]: Changing the copy to v13 so it's consistent.  Look and feel of the Test Replay content will be a bit adjusted from Jennifer's comment, as well

File: docs/guides/cloud/free-trial.mdx (hunk 3 above)
Comment (danjohansenconsulting) [reply docs/guides/cloud/free-trial.mdx:37]: no.  :)

**Review by jennifer-shehane — APPROVED** [review]

**Pull request merged**